packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -1979,7 +1979,7 @@ describe("Teal Opcodes", function () {
         op.execute(stack);
 
         assert.equal(1, stack.length());
-        assert.deepEqual(TXN_OBJ.apat[1], stack.pop());
+        assert.deepEqual(TXN_OBJ.apat[0], stack.pop());
```

Comment:
why are doing this ?
i think `const op = new Txna(["Accounts", "1"], 1, interpreter);` means second account.

Reply:
I though so too. But `Txna Account 0` means sender's address, and the above means first account. 
Tested (on blockchain) using: `gtxn 0 Accounts 1` (this pushed the first account from the account array). Something seems wrong. Will confirm on discord as well.

Reply:
Update: Confimed `const op = new Txna(["Accounts", "1"], 1, interpreter);` means first account.

Will do this in a new PR first, with more tests. We also need to update `Txn`, `Gtxn` as well.

Reply:
i think we already have tests but they are written according to code.
or you can update on this PR as well, In interpreter file we already have `getAccount` function.
update lines with this.getAccount(1,line).your call

Reply:
Yes, but we need to add "more" tests with the correct behaviour, since we previously assumed wrong one, according to which the tests were written (your initial comment). 
Also found out that `Txn`, `Gtxn` needs to be updated as well because they also support array arguments. (eg. `txn Accounts 0`, `gtxn 0 Accounts 0` also works), so it's better to handle these fixes along with this fix in a new PR.

Reply:
like i said, your call